utils/build-script-impl

```diff
@@ -2265,17 +2440,17 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARG
                 if [[ -z "${INSTALL_LLDB}" ]] ; then
                     continue
                 fi
-                case "$(uname -s)" in
-                    Linux)
+                case ${host} in
+                    linux-*)
                         ;;
-                    FreeBSD)
+                    freebsd-*)
                         ;;
-                    CYGWIN_NT-10.0)
+                    cygwin-*)
                         ;;
-                    Darwin)
+                    macosx-*)
                         set_lldb_build_mode
                         pushd ${LLDB_SOURCE_DIR}
-                        xcodebuild -target toolchain -configuration ${LLDB_BUILD_MODE} install ${lldb_xcodebuild_options[@]} DSTROOT="${INSTALL_DESTDIR}" LLDB_TOOLCHAIN_PREFIX="${TOOLCHAIN_PREFIX}"
+                        xcodebuild -target toolchain -configuration ${LLDB_BUILD_MODE} install ${lldb_xcodebuild_options[@]} DSTROOT="${host_install_destdir}"
```

Comment:
> That means `INSTALL_PREFIX` has to include `TOOLCHAIN_PREFIX`, which isn't documented anywhere.

It is true that it is not documented, but that's how existing presets work.  (If we agree to change behavior we would need to update all of them.)

> Also, it's problematic because we would then need host-specific install prefixes when cross-compiling (which is probably a good idea

Sure.

> We can avoid it for now if all hosts use /usr and we insert the toolchain prefix in the build script

As long as existing presets continue to work, I'd be OK with it; although I remember someone has posted toolchains for Linux that install into `/usr/local`, so eventually we'd like to get this configurable again.